src/modules/fancyzones/lib/Settings.cpp

```diff
@@ -133,104 +137,132 @@ IFACEMETHODIMP_(void) FancyZonesSettings::SetConfig(PCWSTR serializedPowerToysSe
     }
     Trace::SettingsChanged(m_settings);
 }
-CATCH_LOG();
 
-IFACEMETHODIMP_(void) FancyZonesSettings::CallCustomAction(PCWSTR action) noexcept try
+IFACEMETHODIMP_(void) FancyZonesSettings::CallCustomAction(PCWSTR action) noexcept
 {
-    // Parse the action values, including name.
-    PowerToysSettings::CustomActionObject action_object =
-        PowerToysSettings::CustomActionObject::from_json_string(action);
+    try
+    {
+        // Parse the action values, including name.
+        PowerToysSettings::CustomActionObject action_object =
+            PowerToysSettings::CustomActionObject::from_json_string(action);
 
-    if (m_callback && action_object.get_name() == NonLocalizable::ToggleEditorActionID)
+        if (m_callback && action_object.get_name() == NonLocalizable::ToggleEditorActionID)
+        {
+            m_callback->ToggleEditor();
+        }
+    }
+    catch (...)
     {
-        m_callback->ToggleEditor();
+        // TODO: Currently only custom action comming from main PowerToys window is request to launch editor.

```

Comment:
@enricogior what is Your opinion on this one ?

Reply:
Let's keep it simple, we only say "Cannot launch FancyZones editor. Please report etc."
So users that will encounter this bug will report it, otherwise they might just use the hotkey and never report it.

Reply:
OK, I'll also leave a note here, that if we add new custom action (if ever), error message should be modified.

Reply:
Done.